src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py

```diff
@@ -176,7 +176,7 @@ def test_keyvault_private_endpoint_connection(self, resource_group):
                  '--description "{approval_desc}"', checks=[
                      self.check('privateLinkServiceConnectionState.status', 'Approved'),
                      self.check('privateLinkServiceConnectionState.description', '{approval_desc}'),
-                     self.check('provisioningState', 'Succeeded')
+                     self.check('provisioningState', 'Updating')
```

Comment:
It looks like the state could be both:
```suggestion
                     self.check_pattern('provisioningState', '(?:Updating|Succeeded)')
```

Reply:
Interesting, how does it work? `(?:Updating|Succeeded)`